### ISSUE TITLE
remove status code

### ISSUE THREAD

**Issue by keldu** [keldu/mc_bot#20]
Not really needed and I don't really support the idea of using this since you could use this to spy on servers and track users on badly configured servers.

**Comment by keldu** [comment]
Realized status code doesn't describe the issue. I want to remove the the sources that build `minecraft_status`, but it's blocked by another issue. Already removed from the build system.
